apps/zero-api/src/users/users.controller.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  UsePipes,
+  ValidationPipe,
+  UseInterceptors,
+  ClassSerializerInterceptor,
+  UseFilters
+} from '@nestjs/common';
+import { UsersService } from './users.service';
+import { CreateUserDto } from './dto/create-user.dto';
+import { ApiCreatedResponse, ApiOkResponse } from '@nestjs/swagger';
+import { UserEntity } from './entities/user.entity';
+import { NoDataInterceptor } from '../interceptors/NoDataInterceptor';
+import { PrismaClientExceptionFilter } from '../exception-filters/PrismaClientExceptionFilter';
+
+@Controller('users')
+@UsePipes(ValidationPipe)
+@UseInterceptors(ClassSerializerInterceptor, NoDataInterceptor)
+@UseFilters(PrismaClientExceptionFilter)
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Post()
+  @ApiCreatedResponse({ type: UserEntity })
+  async create(@Body() createUserDto: CreateUserDto) {
+    return await this.usersService.create(createUserDto);
+  }
+
+  // @Get()
```

Comment:
Please uncomment all this code

Reply:
implemented updating user entity, removed all commented out code that is not required now